frontend/src/features/accounts/invite-form.ts

```diff
@@ -85,6 +83,17 @@ export class InviteForm extends LiteElement {
             )}
           </sl-select>
         </div>
+        <div class="mb-5">
+          <sl-select
+            label=${msg("Role")}
+            value=${AccessCode.crawler}
```

Comment:
Neat, I think this is all that's needed for that?

```suggestion
            value=${AccessCode.owner}
```